Correct `container path` and `cloud` in azure cloud integration (#308)
opencost/opencost-website: docs/configuration/azure.mdx

* `<STORAGE_ACCOUNT>` is the name of the Storage account where the exported Azure cost report data is being stored.
* `<STORAGE_ACCESS_KEY>` can be found by selecting *Access Keys* from the navigation sidebar then selecting Show keys. Using either of the two keys will work.
* `<STORAGE_CONTAINER>` is the name that you chose for the exported cost report when you set it up. This is the name of the container where the CSV cost reports are saved in your Storage account.

> * `<CONTAINER_PATH>` should be passed empty if you do not have one and used only if there is more than one billing report that is exported to the configured container. The path provided should have only one billing export because OpenCost will retrieve the most recent billing report for a given month found within the path.

Reviewer: ```suggestion
* `<CONTAINER_PATH>` should be used if there is more than one billing report that is exported to the configured container. The path provided should have only one billing export because OpenCost will retrieve the most recent billing report for a given month found within the path. If this configuration is not used, it should be set to an empty string `""`.
```

Thoughts on the wording change?

* `<CLOUD>` is the value which denotes the cloud where the storage account exists. Possible values are `public` and `gov`. The default is `public` if an empty string is provided.

> Set these values into the to the Azure array in the `cloud-integration.json`:

Reviewer: ```suggestion
Set these values to the Azure array in the `cloud-integration.json` file:
```

While we're here, may as well fix this grammar issue.
